src/site/content/en/blog/use-long-term-caching/index.md

```diff
@@ -0,0 +1,783 @@
+        // will be included into this chunk
+        minChunks: Infinity,
+    }),
+    ],
+};
+```
+
+After these changes, each build will be generating three files:
+
+```shell
+$ webpack
+Hash: ac01483e8fec1fa70676
+Version: webpack 3.8.1
+Time: 3816ms
+                            Asset     Size  Chunks             Chunk Names
+   ./main.00bab6fd3100008a42b0.js    82 kB       0  [emitted]  main
+ ./vendor.26886caf15818fa82dfa.js    46 kB       1  [emitted]  vendor
+./runtime.79f17c27b335abc7aaf4.js  1.45 kB       3  [emitted]  runtime
+```
+
+Include them into `index.html` in the reverse order – and you’re done:
+
+```html
+<!-- index.html -->
+<script src="./runtime.79f17c27b335abc7aaf4.js"></script>
+<script src="./vendor.26886caf15818fa82dfa.js"></script>
+<script src="./main.00bab6fd3100008a42b0.js"></script>
+```
+
+### Further reading
+
+* Webpack guide [on long term caching](https://webpack.js.org/guides/caching/)
+
+* Webpack docs [about webpack runtime and
+  manifest](https://webpack.js.org/concepts/manifest/)
+
+* [“Getting the most out of the
+  CommonsChunkPlugin”](https://medium.com/webpack/webpack-bits-getting-the-most-out-of-the-commonschunkplugin-ab389e5f318)
+
+* [How `optimization.splitChunks` and `optimization.runtimeChunk` work](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693)
+
+## Inline webpack runtime to save an extra HTTP request
+
+To make things even better, try inlining the webpack runtime into the HTML
+response. I.e., instead of this:
+
+```html
+<!-- index.html -->
+<script src="./runtime.79f17c27b335abc7aaf4.js"></script>
+```
+
+do this:
+
+```html
+<!-- index.html -->
+<script>
+!function(e){function n(r){if(t[r])return t[r].exports;…}} ([]);
+</script>
+```
+
+The runtime is small, and inlining it will help you save an HTTP request (pretty
+important with HTTP/1; less important with HTTP/2 but might still play an
+effect).
+
+Here’s how to do it.
+
+### If you generate HTML with the HtmlWebpackPlugin
+
+If you use the
+[HtmlWebpackPlugin](https://github.com/jantimon/html-webpack-plugin) to generate
+an HTML file, the
+[InlineSourcePlugin](https://github.com/DustinJackson/html-webpack-inline-source-plugin)
+is all you need:
+
+```js
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const InlineSourcePlugin = require('html-webpack-inline-source-plugin');
+
+module.exports = {
+    plugins: [
+    new HtmlWebpackPlugin({
+        inlineSource: 'runtime~.+\\.js',
+    }),
+    new InlineSourcePlugin(),
+    ],
+};
+```
+
+### If you generate HTML using a custom server logic
+
+**With webpack 4:**
+
+1. Add the
+    [`WebpackManifestPlugin`](https://github.com/danethurber/webpack-manifest-plugin)
+    to know the generated name of the runtime chunk:
+
+    ```js
+    // webpack.config.js (for webpack 4)
+    const ManifestPlugin = require('webpack-manifest-plugin');
+
+    module.exports = {
+    plugins: [
+        new ManifestPlugin(),
+    ],
+    };
+    ```
```

Comment:
```suggestion
    ```js
    // webpack.config.js (for webpack 4)
    const ManifestPlugin = require('webpack-manifest-plugin');

    module.exports = {
      plugins: [
        new ManifestPlugin()
      ]
    };
    ```
```